### ISSUE TITLE
Unexpected appearence: \WPy64-3940\scripts

### ISSUE THREAD

**Issue by VSZ2020** [winpython/winpython#993]
Hello! 
I have one trouble with WinPython. I can't launch any program in major programm folder. But I can run Command Prompt. After cmd prompt started I'm getting the following error: "Unexpected appearance: \WPy64-3940\scripts.". I have the simular winpython folder in root directory of my harddrive (WPy64-3910) and there are no troubles with that python version (3.9.1.0). How can I fix the problem with starting of new winpython versions? Could you help me?

**Comment by VSZ2020** [comment]
It was strange, but after moving WPy64-3940 dir into root drive folder, the command prompt starts to work

**Comment by stonebig** [comment]
hum, is there any "<space>" or other odd character in your pasth to winpython ?

**Comment by VSZ2020** [comment]
I solved the problem by moving winPython folder into another working folder

**Comment by VSZ2020** [comment]
> hum, is there any `<space>` or other odd character in your path to winpython ?

No, there is not illegal characters. The problem would be in corrupted windows directory name.
